pkg/ddc/juicefs/status.go

```diff
@@ -68,10 +68,10 @@ func (j *JuiceFSEngine) CheckAndUpdateRuntimeStatus() (ready bool, err error) {
 		runtimeToUpdate.Status.CacheStates[common.CacheCapacity] = states.cacheCapacity
 
 		// Todo:show this infomathion when complete dataload function
-		//runtimeToUpdate.Status.CacheStates[common.CachedPercentage] = states.cachedPercentage
-		//runtimeToUpdate.Status.CacheStates[common.Cached] = states.cached
-		runtimeToUpdate.Status.CacheStates[common.CachedPercentage] = "N/A"
-		runtimeToUpdate.Status.CacheStates[common.Cached] = "N/A"
+		runtimeToUpdate.Status.CacheStates[common.CachedPercentage] = states.cachedPercentage
+		runtimeToUpdate.Status.CacheStates[common.Cached] = states.cached
```

Comment:
I think cache status should set "N/A" in community edition here, because community edition has no distributed cache yet. It's meaningless to show cache status in community edition.

Reply:
I thought "N/A" is confused.
When using community edition, before the fuse pod running the cache state is as shown below.
```
NAME        UFS TOTAL SIZE   CACHED     CACHE CAPACITY   CACHED PERCENTAGE   PHASE   AGE
jfscedemo   37.92GiB                                                         Bound   8h
```
What we want is that after dataload finish, the worker will cache the data we want. When app pod is applying, it should be scheduled to the node in which the worker is running, so that the app pod can use the cache built by worker. As we can use dataload in community edition, and the cache can be reuse by the app pod, cache status is meaningful and absolutely concerned by user. The dataset status is as shown below which is more humanized.
```
NAME        UFS TOTAL SIZE   CACHED     CACHE CAPACITY   CACHED PERCENTAGE   PHASE   AGE
jfscedemo   37.92GiB         37.96GiB   100.00GiB        100.1%              Bound   8h
```

Reply:
ok. got it.